tests/test_meshtastic_utils_async.py

```
@@ -809,7 +810,7 @@ def test_wait_for_future_result_with_shutdown_aborts_when_shutting_down():
     future = MagicMock()
 
     with patch.object(mu, "shutting_down", True):
-        with pytest.raises(TimeoutError, match="Shutdown in progress"):
+        with pytest.raises(FutureWaitShutdownError, match="Shutdown in progress"):
```

Ruff (0.16.0)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)